package.json

```
@@ -4,7 +4,9 @@
   "description": "Adobe XD CLI Plugin Manager Utility",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint --fix lib && eslint --fix commands",
```

1. `lint` script should only validate code, not modify
2. list of folders to lint can be passed one-by-one

```suggestion
    "lint": "eslint lib/** commands/**",
    "lint:fix": "eslint --fix lib/** commands/**",
```